### PR TITLE
quotes: upto 678

### PR DESCRIPTION
**Pull request by AterAnimAvis**
- [x] quote 629 spoiler css
- [x] validate user information again
  - [x] Avatars
  - [x] Nicknames
  - [x] Roles
- [x] Add `nsfw` info for quotes 568+

I've got some cleanup / improvements that I'll continue working on before pr'ing seperately

### CODE REVIEW

File: quotes/data.json [hunk 1]
```diff
@@ -2778,6 +2810,450 @@
...
+            "user": "Amethyst",
+            "text": "we are the curle-borg. you will be bolbed. resistance is futile."
+        },
+        {
+            "user": "SizableShrimp",
+            "text": ":cummies: [...] :yummies:"
+        },
+        {
+            "user": "Amethyst",
+            "text": " he's posting awful curle mechatrons with massive eggplant dicks"
```
Comment (ravenbuilder934) [inline quotes/data.json:2869]: space?

File: quotes/data.json [hunk 2]
```diff
@@ -2778,6 +2810,450 @@
...
+            "user": "Amethyst",
+            "text": "///////////////////7410/*+"
+        },
+        {
+            "user": "Maci",
+            "text": "oh no, a curlmketmkngKNMPDAGSagsdnkjnsadjfnb<AUwehbrnN92u837419498"
+        },
+        {
+            "user": "Amethyst",
+            "text": " the (Party Escort™️ Submission Position™️)™️"
```
Comment (ravenbuilder934) [inline quotes/data.json:2885]: space?

**Comment by AterAnimAvis** [comment]
Only thing left to do here is the NSFW boolean

File: quotes/data.json [hunk 3]
```diff
@@ -2778,6 +2809,450 @@
...
+            "user": "SizableShrimp",
+            "text": "I now shall call you leaky luigi"
+        },
+        {
+            "user": "Amethyst",
+            "text": "i'm the true quote queen in this room <i>flicks hair</i>"
+        },
+        {
+            "user": "Saksham4106",
+            "text": "im going to fuck windows completely now"
```
Comment (ravenbuilder934) [inline quotes/data.json:2852]: nsfw?

File: quotes/data.json [hunk 4]
```diff
@@ -2778,6 +2809,450 @@
...
+            "user": "diesieben07",
+            "text": "shrimp and cyborg-curle fanfic when"
+        },
+        {
+            "user": "Amethyst",
+            "text": "we are the curle-borg. you will be bolbed. resistance is futile."
+        },
+        {
+            "user": "SizableShrimp",
+            "text": ":cummies: [...] :yummies:"
```
Comment (ravenbuilder934) [inline quotes/data.json:2864]: nsfw.

File: quotes/data.json [hunk 5]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "Amethyst",
+            "text": "he's posting awful curle mechatrons with massive eggplant dicks"
```
Comment (ravenbuilder934) [inline quotes/data.json:2868]: nsfw

File: quotes/data.json [hunk 6]
```diff
@@ -2778,6 +2809,450 @@
...
+            "user": "Maci",
+            "text": "oh no, a curlmketmkngKNMPDAGSagsdnkjnsadjfnb<AUwehbrnN92u837419498"
+        },
+        {
+            "user": "Amethyst",
+            "text": "the (Party Escort™️ Submission Position™️)™️"
+        },
+        {
+            "user": "Amethyst",
+            "text": "Aperture can suck my quantum tunneling cock"
```
Comment (ravenbuilder934) [inline quotes/data.json:2888]: nsfw

File: quotes/data.json [hunk 7]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "Amethyst",
+            "text": "Fellate me"
```
Comment (ravenbuilder934) [inline quotes/data.json:2892]: nsfw

File: quotes/data.json [hunk 8]
```diff
@@ -2778,6 +2809,450 @@
...
+            "user": "Maci",
+            "text": "quick, somebody rotate that onion!"
+        },
+        {
+            "user": "Amethyst",
+            "text": "whoops, didn't mean to shark beans there"
+        },
+        {
+            "user": "AterAnimAvis",
+            "text": "It’s only the first CUM, its not boring "
```
Comment (ravenbuilder934) [inline quotes/data.json:2904]: nsfw

File: quotes/data.json [hunk 9]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "AshersLab",
+            "text": "Curle <i>is</i> the only confirmed computer person in the whole of the UK"
+        },
+        {
+            "user": "Saksham4106",
+            "text": "beat it with a dick until it works"
```
Comment (ravenbuilder934) [inline quotes/data.json:2912]: nsfw?

File: quotes/data.json [hunk 10]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "Maci",
+            "side_text": "aka <strong style=\"font-size: 115%\">Fox Apostle Maci</strong>",
+            "text": "wait shit im supposed to be dead"
+        },
+        {
+            "user": "AshersLab",
+            "text": "Yes I'm f***in asher"
```
Comment (ravenbuilder934) [inline quotes/data.json:2921]: nsfw.

File: quotes/data.json [hunk 11]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "diesieben07",
+            "text": "have you tried replacing tit?"
```
Comment (ravenbuilder934) [inline quotes/data.json:2925]: nsfw?

File: quotes/data.json [hunk 12]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "Amethyst",
+            "text": "here's my cum [...] it's thicc"
```
Comment (ravenbuilder934) [inline quotes/data.json:2929]: nsfw.

File: quotes/data.json [hunk 13]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "dpeter99",
+            "side_text": "in response to quote 596",
+            "text": "juicy [...] it is stringy.... [...] now it is in my face"
```
Comment (ravenbuilder934) [inline quotes/data.json:2934]: nsfw?

File: quotes/data.json [hunk 14]
```diff
@@ -2778,6 +2809,450 @@
...
+            "user": "FinnT730",
+            "text": "They UnusAbus twitter still exists"
+        },
+        {
+            "user": "diesieben07",
+            "text": "CANS! CANS! There's NO BABY."
+        },
+        {
+            "user": "Amethyst",
+            "text": "Yes ashers, a can of peas costs 40 penises"
```
Comment (ravenbuilder934) [inline quotes/data.json:2946]: nsfw

File: quotes/data.json [hunk 15]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "Maci",
+            "text": "SIX HUNDRED PENISES"
```
Comment (ravenbuilder934) [inline quotes/data.json:2950]: nsfw

File: quotes/data.json [hunk 16]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "AshersLab",
+            "text": "Penis?"
```
Comment (ravenbuilder934) [inline quotes/data.json:2954]: nsfw

File: quotes/data.json [hunk 17]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "AshersLab",
+            "side_text": "to <strong style=\"font-size: 115%\">Maci</strong>",
+            "text": "Oh, you like other people imagining you and penises?"
```
Comment (ravenbuilder934) [inline quotes/data.json:2959]: nsfw

File: quotes/data.json [hunk 18]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "Saksham4106",
+            "text": "lies, its a gigantic cock"
```
Comment (ravenbuilder934) [inline quotes/data.json:2963]: nsfw

File: quotes/data.json [hunk 19]
```diff
@@ -2778,6 +2809,450 @@
...
+            "user": "Maci",
+            "text": "the more they bounce around the lewder they get"
+        },
+        {
+            "user": "Amethyst",
+            "text": "I'm fucking suffocating you bitch"
+        },
+        {
+            "user": "Amethyst",
+            "text": "Unbe is too deep"
```
Comment (ravenbuilder934) [inline quotes/data.json:2975]: nsfw

File: quotes/data.json [hunk 20]
```diff
@@ -2778,6 +2809,450 @@
...
+            "side_text": "to <strong style=\"font-size: 115%\">Maci</strong>",
+            "text": "Oh, you like other people imagining you and penises?"
+        },
+        {
+            "user": "Saksham4106",
+            "text": "lies, its a gigantic cock"
+        },
+        {
+            "user": "Maci",
+            "text": "the more they bounce around the lewder they get"
```
Comment (ravenbuilder934) [inline quotes/data.json:2967]: nsfw?

File: quotes/data.json [hunk 21]
```diff
@@ -2778,6 +2809,450 @@
...
+            "user": "Amethyst",
+            "text": "I'm fucking suffocating you bitch"
+        },
+        {
+            "user": "Amethyst",
+            "text": "Unbe is too deep"
+        },
+        {
+            "user": "Amethyst",
+            "text": "any hole's a goal"
```
Comment (ravenbuilder934) [inline quotes/data.json:2979]: nsfw?

File: quotes/data.json [hunk 22]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "Amethyst",
+            "text": "lol president for porn"
```
Comment (ravenbuilder934) [inline quotes/data.json:2983]: nsfw

File: quotes/data.json [hunk 23]
```diff
@@ -2778,6 +2809,450 @@
...
+            "side_text": "to <strong style=\"font-size: 115%\">Finn</strong>",
+            "text": "I will try you [..] right now"
+        },
+        {
+            "user": "diesieben07",
+            "text": "THE THIS CAT"
+        },
+        {
+            "user": "Amethyst",
+            "text": "reject humanity, return to mnke"
```
Comment (ravenbuilder934) [inline quotes/data.json:3008]: ..typo?

File: quotes/data.json [hunk 24]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "diesieben07",
+            "text": "Okay, enough lewd foxes for today. Good night"
+        },
+        {
+            "user": "Lex",
+            "text": "All current IDEs suck, we need a new one.. with blackjack.. and hookers..."
```
Comment (ravenbuilder934) [inline quotes/data.json:3016]: nsfw?

File: quotes/data.json [hunk 25]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "ChampionAsh5357",
+            "text": "Gwen, what is a potato?"
+        },
+        {
+            "user": "SizableShrimp",
+            "side_text": "aka <strong style=\"font-size: 115%\">SizableShirmp</strong>",
+            "text": "You're not wrong but we don't need to talk about curle's asshole"
```
Comment (ravenbuilder934) [inline quotes/data.json:3025]: nsfw?

File: quotes/data.json [hunk 26]
```diff
@@ -2778,6 +2809,450 @@
...
+            "text": "#GiantBinaryOperatorsIsSexyAndYouKnowIt"
+        },
+        {
+            "user": "Amethyst",
+            "text": "raven asserting dominance over the native asher slab"
+        },
+        {
+            "user": "Amethyst",
+            "side_text": "aka <strong style=\"font-size: 115%\">CurlieWurlie</strong>",
+            "text": "oh god has illy been fucking his cat again"
```
Comment (ravenbuilder934) [inline quotes/data.json:3042]: nsfw.

File: quotes/data.json [hunk 27]
```diff
@@ -2778,6 +2809,450 @@
...
+            "user": "Lex",
+            "text": "oh you're writing mixins? no fuck off"
+        },
+        {
+            "user": "Maci",
+            "text": "<i>destroy me</i>"
+        },
+        {
+            "user": "Maci",
+            "text": "Raven is octopussy confirmed"
```
Comment (ravenbuilder934) [inline quotes/data.json:3054]: nsfw?

File: quotes/data.json [hunk 28]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "Maci",
+            "text": "Seduce me"
```
Comment (ravenbuilder934) [inline quotes/data.json:3058]: nsfw?

File: quotes/data.json [hunk 29]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "AterAnimAvis",
+            "text": "Butt, Butt [...] Big Big Praise [...] It's fucking hard"
+        },
+        {
+            "user": "AterAnimAvis",
+            "text": "<span class=\"spoiler\">Incest in a Private localized livingroom son™ (only 5 small payments of 59.999.999,99)</span>"
```
Comment (ravenbuilder934) [inline quotes/data.json:3066]: REALLY fucking nsfw

File: quotes/data.json [hunk 30]
```diff
@@ -2778,6 +2809,450 @@
...
+            "user": "Maci",
+            "text": "Raven is octopussy confirmed"
+        },
+        {
+            "user": "Maci",
+            "text": "Seduce me"
+        },
+        {
+            "user": "AterAnimAvis",
+            "text": "Butt, Butt [...] Big Big Praise [...] It's fucking hard"
```
Comment (ravenbuilder934) [inline quotes/data.json:3062]: nsfw?

File: quotes/data.json [hunk 31]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "AterAnimAvis",
+            "text": "<span class=\"spoiler\">Incest in a Private localized livingroom son™ (only 5 small payments of 59.999.999,99)</span>"
+        },
+        {
+            "user": "FinnT730",
+            "text": "were squirrel fucks are given to each of us"
```
Comment (ravenbuilder934) [inline quotes/data.json:3070]: nsfw?

File: quotes/data.json [hunk 32]
```diff
@@ -2778,6 +2809,450 @@
...
+            "user": "Amethyst",
+            "text": "The daddy is into mummification? I'd stay away, personally"
+        },
+        {
+            "user": "Amethyst",
+            "text": "I've always been philosophical, i just hide it behind a curtain of memery"
+        },
+        {
+            "user": "Citrine",
+            "text": "curle can go very deep when she wants to"
```
Comment (ravenbuilder934) [inline quotes/data.json:3082]: nsfw?

File: quotes/data.json [hunk 33]
```diff
@@ -2778,6 +2809,450 @@
...
+            "user": "illy",
+            "text": "Yar Shiver me compilers"
+        },
+        {
+            "user": "Maci",
+            "text": "peas are just compiled beans"
+        },
+        {
+            "user": "AterAnimAvis",
+            "text": "You fucked and then saw it coming?"
```
Comment (ravenbuilder934) [inline quotes/data.json:3106]: nsfw.

File: quotes/data.json [hunk 34]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "Maci",
+            "text": "yarrrrr ya never saw it cummin'"
```
Comment (ravenbuilder934) [inline quotes/data.json:3110]: nsfw.

File: quotes/data.json [hunk 35]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "Maci",
+            "text": "how dare you fuck someone and see them at the same time"
```
Comment (ravenbuilder934) [inline quotes/data.json:3114]: nsfw.

File: quotes/data.json [hunk 36]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "Amethyst",
+            "text": "illy is cpw's sub now?"
```
Comment (ravenbuilder934) [inline quotes/data.json:3118]: nsfw?

File: quotes/data.json [hunk 37]
```diff
@@ -2778,6 +2809,450 @@
...
+            "user": "Amethyst",
+            "text": "Hold it in [...] <del>it'll feel much better at the end</del>"
+        },
+        {
+            "user": "Citrine",
+            "text": "do me! do me!"
+        },
+        {
+            "user": "AterAnimAvis",
+            "text": "I'll grab the lube"
```
Comment (ravenbuilder934) [inline quotes/data.json:3130]: nsfw?

File: quotes/data.json [hunk 38]
```diff
@@ -2778,6 +2809,450 @@
...
+            "user": "Amethyst",
+            "text": "if we're not at 700 [quotes] by the end of the day all of you are banned"
+        },
+        {
+            "user": "Maci",
+            "text": "Quotes are like Peas, theyre good for throwing at curle"
+        },
+        {
+            "user": "Maci",
+            "text": "ah yes [..] fellation [..] the classic family activity"
```
Comment (ravenbuilder934) [inline quotes/data.json:3142]: nsfw.

File: quotes/data.json [hunk 39]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "AterAnimAvis",
+            "text": "You have to admit its a nice rack"
```
Comment (ravenbuilder934) [inline quotes/data.json:3146]: nsfw, maybe?

File: quotes/data.json [hunk 40]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "Maci",
+            "text": "massive rack and a butt big enough to have its own phone number"
```
Comment (ravenbuilder934) [inline quotes/data.json:3150]: nsfw, maybe? prolly not but *shrugs*

File: quotes/data.json [hunk 41]
```diff
@@ -2778,6 +2809,450 @@
...
+            "user": "gigaherz",
+            "text": "imagine having free hamburgers available any time. hungry leaving work? no matter, here's a hamburger. homeless? no matter, <img class=\"emote\" src=\"https://discordapp.com/assets/3a2a5ebb52c26a470f87b1eb42c0da92.svg\" alt=\":hamburger:\"/> for you too!"
+        },
+        {
+            "user": "AshersLab",
+            "text": "I'm the settings"
+        },
+        {
+            "user": "Amethyst",
+            "text": "Calculus makes finn hard?"
```
Comment (ravenbuilder934) [inline quotes/data.json:3182]: nsfw?

File: quotes/data.json [hunk 42]
```diff
@@ -2778,6 +2809,450 @@
...
+            "user": "illy",
+            "text": "Can't spell silly without illy"
+        },
+        {
+            "user": "Maci",
+            "text": "dogs and cats count as foxes"
+        },
+        {
+            "user": "Saksham4106",
+            "text": "finn's sex dungeon"
```
Comment (ravenbuilder934) [inline quotes/data.json:3194]: nsfw.

File: quotes/data.json [hunk 43]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "SizableShrimp",
+            "text": "boy ass juicy"
```
Comment (ravenbuilder934) [inline quotes/data.json:3198]: nsfw?

File: quotes/data.json [hunk 44]
```diff
@@ -2778,6 +2809,450 @@
...
+            "user": "Anonymous",
+            "text": "DRIPPING FLESHY BLOODGORE FESTERS IN YOUR GAPING WOUNDS! DEMONS 'N' STUFF! GRAAAAR!"
+        },
+        {
+            "user": "Amethyst",
+            "text": "C.... Cheese?"
+        },
+        {
+            "user": "SizableShrimp",
+            "text": "purity = cock??"
```
Comment (ravenbuilder934) [inline quotes/data.json:3222]: nsfw?

File: quotes/data.json [hunk 45]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "SizableShrimp",
+            "text": "get. fucked. raven"
```
Comment (ravenbuilder934) [inline quotes/data.json:3226]: nsfw?

File: quotes/data.json [hunk 46]
```diff
@@ -2778,6 +2809,450 @@
...
+        },
+        {
+            "user": "Amethyst",
+            "text": "By popular demand, Forge has grown a CoC!"
```
Comment (ravenbuilder934) [inline quotes/data.json:3230]: nsfw?

File: quotes/data.json [hunk 47]
```diff
@@ -2778,6 +2809,450 @@
...
+            "text": "I like this quote way too much"
+        },
+        {
+            "user": "Amethyst",
+            "side_text": "aka <strong style=\"font-size: 115%\">Curel</strong>",
+            "text": "ain't gonna prevent much birth if it got a hole"
+        },
+        {
+            "user": "SizableShrimp",
+            "text": "WHERE'S THE DICK"
```
Comment (ravenbuilder934) [inline quotes/data.json:3243]: nsfw?

**Review by ravenbuilder934 — CHANGES_REQUESTED** [review]
NSFW boolean stuff. if I was on the fence I tried to mark it as nsfw (but with a question mark). some existing ones may need nsfw markings as well:
 - 7
 - 42(?)
 - 68(?)
 - 76
 - 77(?)
 - 81(?)
 - 91
 - 124(?)
 - 164(?)
 - 177(?)
 - 181(?)
 - 195(?)
 - 208, 210, 211?
 - 221(?)
 - Remove 226, maybe
 - 235
 - 238, 239(?)
 - 240
 - 245, 249(?)
 - 269(?)
 - 273(?)
 - 274
 - 299
 - 300(?)
 - 323(?)
 - 331
 - 348
 - 356
 - 380(?)
 - 385(?)
 - 399(?)
 - 407(?)
 - 451(?)
 - 457
 - 458
 - 459
 - 467(?)
 - 468(?)
 - 476(?)
 - 480
 - 499(?)
 - 515
 - 536(?)
 - 541(?)
 - 542(?)
 - 562(?)

**Review by ravenbuilder934 — APPROVED** [review]

**Review by sciwhiz12 — APPROVED** [review]
It is time. _(and i just remembered about this)_
:shipit:

**Pull request merged**